system-test/bigquery.ts

```diff
@@ -176,7 +176,6 @@ describe('BigQuery', () => {
       filter: `labels.${GCLOUD_TESTS_PREFIX}`,
     });
     assert(datasets[0] instanceof Dataset);
-    assert.strictEqual(nextQuery, null);
```

Comment:
seems reasonable to me; in a perfect world, I might try to have a `beforeEach` that ensures that there's nothing populated in `nextQuery` before running the test. Is there a tracking issue for this bug? (state between tests can really start to be a pain).

Reply:
I don't think there is, @JustinBeckwith might know though?